packages/lesswrong/components/community/modules/CommunityMembers.tsx

```diff
@@ -252,8 +263,8 @@ const CommunityMembers = ({currentUser, userLocation, distanceUnit='km', locatio
             <div className={classes.location}>{hit.mapLocationAddress}</div>
           </div>
         </div>
-        {hit.htmlBio && <ContentStyles contentType="comment" className={classes.description}>
-          <div dangerouslySetInnerHTML={{__html: truncate(hit.htmlBio, 220)}} />
+        {hit.bio && <ContentStyles contentType="comment" className={classes.description}>
+          {hit.bio}
```

Comment:
I noticed htmlBio wasn't in the results anymore. Seems fine to use bio instead so I change this.